images/custom-scorecard-tests/cmd/test/main.go

```diff
@@ -53,10 +52,10 @@ func main() {
 	// Names of the custom tests which would be passed in the
 	// `operator-sdk` command.
 	switch entrypoint[0] {
-	case tests.CustomTest1Name:
-		result = tests.CustomTest1(cfg)
-	case tests.CustomTest2Name:
-		result = tests.CustomTest2(cfg)
+	case CustomTest1Name:
+		result = CustomTest1(cfg)
+	case CustomTest2Name:
+		result = CustomTest2(cfg)
```

Comment:
Can we have just 2 custom tests? I did not get this part.

Reply:
this is just an example of a custom image, it has 2 tests that essentially are no-op's to just show the structure of what a custom image looks like, real custom images that an ISV would build would have any number of tests in reality.

Reply:
it has 2 only as an example....a custom image can support any number of tests.